spoe_auth_snap/snapcraft.yaml

```diff
@@ -0,0 +1,40 @@
+name: haproxy-spoe-auth
+title: HAProxy SPOE Authentication Agent
+base: core22
+version: '0.0.1'
+summary: HAProxy SPOE authentication agent
+description: |
+  A SPOE (Stream Processing Offload Engine) agent for HAProxy that provides
+  authentication capabilities.
+
+grade: stable
+confinement: strict
+parts:
+  haproxy-spoe-auth:
+    plugin: dump
+    source: https://github.com/Thanhphan1147/haproxy-spoe-auth.git
```

Comment:
Can you fork into the Canonical org and use that link instead?

Reply:
I'll prepare a PR for upstream and once the PR is merged I'll update this to upstream.